website/pages/docs/recipes/destinations/snowflake.md

```diff
@@ -0,0 +1,18 @@
+# Snowflake Destination Plugin Recipes
+
+Full spec options for Snowflake destination available [here](https://github.com/cloudquery/cloudquery/tree/main/plugins/destination/snowflake).
+
+Note: Make sure you use environment variable expansion in production instead of committing the credentials to the configuration file directly.
+
+## Basic
+
+```yaml copy
+kind: destination
+spec:
+  name: snowflake
+  path: cloudquery/snow
+  version: "v1.0.1" # latest version of postgresql plugin
```

Comment:
```suggestion
  version: "v1.0.1" # latest version of snowflake plugin
```

Reply:
Good catch. My upcoming PR should solve variables over strings hopefully :)